.github/workflows/windows.yml

```diff
@@ -102,6 +102,17 @@ jobs:
             ridk exec patch -p1 -i `
               ..\mroonga\packages\source\patches\mariadb-10.5.5-add-a-missing-export-symbol.diff
           }
+          # mariadb-10.2.36, 10.3.27, 10.4.17, 10.5.8 have a syntax error in CMakeList of mariadb-connector-c.
+          # (mariadb-connector-c is MariaDB client library.)
+          # This bug has already fixed by https://github.com/mariadb-corporation/mariadb-connector-c/commit/242cab8cbcd91af882233730a83627d3b12ced83
+          # However, this fix is not included that above version of mariadb.
+          # Therefore, we fix this bug by using this patch until next release for mariadb.
+          # We will remove this patch when the next version of mariadb will be released.
+          ridk exec curl `
+            -o mariadb-connector-c-fix-syntax-error-in-cmake-3.20.diff `
+            https://github.com/mariadb-corporation/mariadb-connector-c/commit/242cab8cbcd91af882233730a83627d3b12ced83.patch
+          ridk exec patch -d libmariadb -p1 -i `
+            ..\mariadb-connector-c-fix-syntax-error-in-cmake-3.20.diff
```

Comment:
Could you use the downloaded patch?

Reply:
`mariadb-connector-c-fix-syntax-error-in-cmake-3.20.diff` that I'm using here is the downloaded patch.

It is renamed to `mariadb-connector-c-fix-syntax-error-in-cmake-3.20.diff` from `242cab8cbcd91af882233730a83627d3b12ced83.patch` .

Should I use the original patch file name (`242cab8cbcd91af882233730a83627d3b12ced83.patch`) ?
Because `mariadb-connector-c-fix-syntax-error-in-cmake-3.20.diff` is confusing.

Reply:
Ah, `..\` is needed because we use `-d`.
I understand.